livekit-agents/livekit/agents/ipc/job_main.py

```diff
@@ -47,6 +47,10 @@ def _forward_logs(self):
 
     def emit(self, record: logging.LogRecord) -> None:
         try:
+            # Check if Python is shutting down
+            if not hasattr(sys, "meta_path") or sys.meta_path is None:
+                return
+
```

Comment:
Would [sys.is_finalizing()](https://docs.python.org/3/library/sys.html#sys.is_finalizing) do the work instead?

Reply:
Indeed! Fixed!